src/components/List/Section.js

```diff
@@ -29,8 +28,8 @@ const TitleText = styled(BoldText)`
   text-transform: uppercase;
 `;
 
-const Section = ({ title, titleStyle, children, ...rest }) => (
-  <SectionContainer {...rest}>
+const Section = ({ title, titleStyle, children, container }) => (
+  <SectionContainer style={{ ...container }}>
```

Comment:
Is this change really necessary? Couldn't you pass through a `style` prop with the current implementation to override the `styled-component`'s styles?